source/includes/_dataset_metadata.md

```diff
@@ -26,14 +30,44 @@ This object stores both the definition and the value of a given metadata.
 Attribute | Description
 --------- | -----------
 `name` <br> *string* | Identifier for the object (inherited from the [definition](#the-metadata-definition-object)'s name)
-`template` <br> *string* | Identifier for the group of metadata
+`template` <br> *[metadate template object](#the-metadata-template-object)* | Group of the current object
 `definition` <br> *[form object](#the-form-object)* <br> <em class="expandable">expandable</em> | The definition of the metadata type and widget
 `value` | The object's value (may not be the indexed value, see below)
 `remote_value` | The remote object's metadata value (see below)
 `override_remote_value` <br> *boolean* | Flag indicating whether the indexed value is `value` or `remote_value`
 
 In the case of federated and harvested datasets, metadata values are automatically collected from the remote source, showing up in the object as `remote_value`. You can however override this value with your own, specifying it in `value` and setting the `override_remote_value` flag to `True`. This flag determines which value will show up in the explore API output.
 
+## The metadata template object
+
+> Example object
+
+```json
+{
+    "name": "default",
+    "title": "Default metadata",
```

Comment:
The weird part there is "basic" :) Namespace is too technical and not descriptive enough of the reason why this needs to exist. I say purpose and we rename basic into general/generic